app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -334,6 +358,11 @@ public boolean onContextItemSelected(MenuItem item) {
             case R.id.remove_item:
                 RemoveFeedDialog.show(getContext(), feed, null);
                 return true;
+            case R.id.multi_select:
+                speedDialView.setVisibility(View.VISIBLE);
+                if (subscriptionAdapter.onContextItemSelected(item)) {
+                    return true;
+                }
```

Comment:
It could use some refactoring. But currently it is needed by the adapter to know when multi select is selected. So yes, it is deliberate

Reply:
The adapter is already called explicitly. Why is it needed to fall through into the default branch and execute `super.onContextItemSelected(item)` when the adapter returns `false`?

Reply:
Because i put the onContextItemSelected logic for  mulitselect in adapter. Not sure if that is a good idea. That is why i have considered refactoring.